src/Symfony/Component/Ldap/Adapter/ExtLdap/EntryManager.php

```diff
@@ -69,10 +69,7 @@ public function remove(Entry $entry)
     }
 
     /**
-     * Adds values to an entry's multi-valued attribute from the LDAP server.
-     *
-     * @throws NotBoundException
-     * @throws LdapException
+     * {@inheritdoc}
```

Comment:
let's keep the `@throws` exception for now since they're not on the interface yet.